code/modules/projectiles/guns/projectile/random_pistol.dm

```diff
@@ -58,12 +44,24 @@
 		ICON_STATE_WORLD = list(17, 15),
 		ICON_STATE_INV = list(17, 17)
 	)
+	var/list/handle_icons = list(
```

Comment:
static/global?

Reply:
it's a decl anyway, and in theory can be overriden for different looks sets